### PR TITLE
feat(GuidesLink): updates Links to xDSL Guides

### PR DESCRIPTION
**Pull request by Steffy29**
Updates Links to xDSL Guides on the Telecom Dashboard page

close #XDSL-5046

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-manager#56.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✔ 
  * Build manager US ✔ 
  * Tests ✔ 
* Stage 3
  * Prepare tar.gz for UXCI / Release ✘

**Comment by antleblanc** [comment]
You can omit emojis from your commit message and please update only `Messages_fr_FR.xml` files as well.

File: client/app/telecom/dashboard/translations/Messages_fi_FI.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
    <translation id="telecom_dashboard_FAQ_Activate" qtlid="305018">Palvelujen aktivointi</translation>
    <translation id="telecom_dashboard_FAQ_managed_modem" qtlid="305031">Modeemin konfigurointi</translation>
    <translation id="telecom_dashboard_FAQ_init_modem" qtlid="305044">Modeemin nollaus</translation>
+   <translation id="telecom_dashboard_FAQ_restore_service">Palauta palvelusi</translation>
```
Comment (euhmeuh) [inline client/app/telecom/dashboard/translations/Messages_fi_FI.xml]: You don't have to send the translations, our translator team will handle everything. Plus, those files will be erased by our translation system when building, so there's no need to provide other texts than French.

File: client/app/telecom/dashboard/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -1,48 +1,55 @@
...
+    <translation id="telecom_dashboard_auth_failed" qtlid="305200">Impossible de récupérer les informations de
+        l'utilisateur.
+    </translation>
+    <translation id="telecom_dashboard_express_lite_desc" qtlid="455712">Express Lite comprend donc un modem offrant un
+        accès à internet et une ligne VoIP pour <span class="ad-price font-weight-bold fs14">22,99€ HT</span>&#160;/&#160;mois.
+    </translation>
+    <translation id="telecom_dashboard_express_lite_title" qtlid="304888">OVH lance une nouvelle offre de connexion ADSL
+        et VDSL incluant l'essentiel.
+    </translation>
+    <translation id="telecom_dashboard_discover" qtlid="304927">Découvrir l'offre</translation>
```
Comment (euhmeuh) [inline client/app/telecom/dashboard/translations/Messages_fr_FR.xml]: Did you change something apart from the indentation? It's harder to notice :/
Maybe we should revert this change and handle the indentation in another fix.
